internal/proxy/powerflex_handler.go

```diff
@@ -907,6 +912,101 @@ func (s *System) volumeUnmapHandler(next http.Handler, enf *quota.RedisEnforceme
 	})
 }
 
+func (s *System) sdcApproveHandler(next http.Handler, sdcapp *sdc.RedisSdcApprover, opaHost string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ctx, span := trace.SpanFromContext(r.Context()).TracerProvider().Tracer("").Start(r.Context(), "sdcApproveHandler")
+		defer span.End()
+
+		// Read the body.
+		b, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			writeError(w, "powerflex", "failed to read body", http.StatusInternalServerError, s.log)
+			return
+		}
+		defer r.Body.Close()
+
+		var requestBody map[string]json.RawMessage
```

Comment:
It doesn't look like the requestBody is used or modified in this handler. If we're not doing that, we don't need to read it.

Reply:
On a second look, we are doing this in other handler so it's ok to leave this. We will have the request body ready for the future incase we need to modify it.

Reply:
Okay. Left this